[DOCS] Add OOO Messaging to GX Cloud Get Support (#9100)
great-expectations/great_expectations: docs/docusaurus/docs/cloud/about_gx.md

## Get support

:::note Support during the holiday season

> Customer support is available from December 25 to December 29, 2023. If you request support after December 29, the Developer Relations team will respond when they return the first week of January 2024.

Reviewer: ```suggestion
Customer support is not available from December 25 to December 29, 2023. If you request support after December 25, we will respond the first week of January 2024.
```
